site-src/geps/gep-1324.md

```diff
@@ -0,0 +1,127 @@
+# GEP-1324: Service Mesh in Gateway API
+
+* Issue: [#1324](https://github.com/kubernetes-sigs/gateway-api/issues/1324)
+* Status: Provisional
+
+## Overview
+
+Gateway API represents the next generation of traffic routing APIs in Kubernetes. While most of the current work for Gateway API is focused on the ingress use-case, support for service mesh implementations within the spec would be advantageous to the greater community. Therefore, this GEP serves as the genesis for developing common patterns for using Gateway API for east/west traffic within a mesh implementation.
+
+## Goals
+
+* Augment the Gateway API specification with the necessary models, resources, and modifications so that the xRoute primitives are useable by service mesh implementations
+* Ensure common policies applicable to both north/south and east/west are respected
```

Comment:
I think we need to elaborate on this one, can we make this goal higher level, or explain it further so there's little or no ambiguity?

Reply:
This goal originated from this Google Docs [comment](https://docs.google.com/document/d/1IzItdX22fHfXCP6Zch6pC8hEImMNJ6MCPqGNW1aaDr8/edit?disco=AAAAeBMkHBw), but reading back through it, I think it's covered by the non-goal around duplicating concerns. So, I feel like this could probably be removed?

Reply:
ok :+1: